src/sentry/runner/commands/cleanup.py

```diff
@@ -226,6 +226,17 @@ def is_filtered(model):
 
             queryset.delete()
 
+    if not silent:
+        click.echo("Removing expired files associated with ExportedData")
+
+    if is_filtered(models.ExportedData):
+        if not silent:
+            click.echo(">> Skipping ExportedData files")
+    else:
+        queryset = models.ExportedData.objects.filter(date_expired__lt=(timezone.now()))
+        for item in queryset:
+            item.file.delete()
```

Comment:
Should this be `item.delete()`? Or do we want to keep the record of the export around?

Reply:
It's definitely safe to delete the actual item, but since the links are emailed to the user, It'd keep throwing 404s if we deleted the record and they clicked old links. If we keep the records, we're gonna show users 'Your page has expired' pages instead